apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -173,17 +173,19 @@ const CurriculumCatalog = ({
         imageUrl={CourseCatalogIllustration01}
       />
       {showAssignSuccessMessage && (
-        <div className={style.assignSuccessMessageContainer}>
-          <BodyTwoText className={style.assignSuccessMessage}>
-            {assignSuccessMessage}
-          </BodyTwoText>
-          <button
-            aria-label="close success message"
-            onClick={handleCloseAssignSuccessMessage}
-            type="button"
-          >
-            <strong>X</strong>
-          </button>
+        <div className={style.assignSuccessMessageCenter}>
```

Comment:
What is the purpose of having the surrounding `assignSuccessMessageCenter` div rather than just updating the sytles of `assignSuccessMessageContainer`?

Reply:
I tried applying styling to `assignSuccessMessageContainer` but was not getting the results I wanted.